llama_index/vector_stores/chroma.py

```diff
@@ -82,6 +83,7 @@ def __init__(
         port: Optional[str] = None,
         ssl: bool = False,
         headers: Optional[Dict[str, str]] = None,
```

Comment:
we should probably add this `persist_dir` as a class attribute like `host`, `port`, and `ssl`.

Reply:
just saw that @Disiok noted the same thing.